IPC_computetrigger.py

```diff
@@ -0,0 +1,189 @@
+import pandas as pd
+from utils import parse_args, parse_yaml, config_logger
+from pathlib import Path
+import logging
+import numpy as np
+import os
+
+logger = logging.getLogger(__name__)
+
+# TODO: check that all cols, so CS ML1 ML2 1 to 5 and pop cols are present in input data
+# TODO: quality check that perc cols add up to 100
+
+
+def add_columns(df, source):
+    df["date"] = pd.to_datetime(df["date"])
+    df["year"] = df["date"].dt.year
+    df["month"] = df["date"].dt.month
+
+    df["Source"] = source
+
+    df = df.rename(columns={"adjusted_population": "pop_Country", "ADM1_EN": "ADMIN1"})
+
+    # calculate percentage of population per status and level
+    for status in ["CS", "ML1", "ML2"]:
+        for i in range(1, 6):
```

Comment:
why range(1,6)? (please include answer in code as comment: eg: # number of IPC index levels)